Examples/Swift/ViewController.swift

```diff
@@ -124,7 +124,6 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
         super.viewWillAppear(animated)
 
         // Reset the navigation styling to the defaults
-        DayStyle().apply()
```

Comment:
This needs to be fixed. This should fire only when the NVC _returns_ from a navigation session.